public/app/plugins/datasource/cloud-monitoring/components/ConfigEditor/ConfigEditor.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { PureComponent } from 'react';
+import { Select, FieldSet, InlineField, Alert } from '@grafana/ui';
+import { DataSourcePluginOptionsEditorProps, onUpdateDatasourceJsonDataOptionSelect } from '@grafana/data';
+import { AuthType, authTypes, CloudMonitoringOptions, CloudMonitoringSecureJsonData } from '../../types';
+import { JWTConfig } from './JWTConfig';
+
+export type Props = DataSourcePluginOptionsEditorProps<CloudMonitoringOptions, CloudMonitoringSecureJsonData>;
+
+export class ConfigEditor extends PureComponent<Props> {
+  render() {
+    const { options, onOptionsChange } = this.props;
+    const { secureJsonFields, jsonData } = options;
+
+    if (!jsonData.hasOwnProperty('authenticationType')) {
+      jsonData.authenticationType = AuthType.JWT;
+    }
+
+    return (
+      <>
+        <div className="gf-form-group">
+          <div className="grafana-info-box">
+            <h4>Google Cloud Monitoring Authentication</h4>
+            <p>
+              There are two ways to authenticate the Google Cloud Monitoring plugin - either by uploading a Service
+              Account key file or by automatically retrieving credentials from the Google metadata server. The latter
+              option is only available when running Grafana on a GCE virtual machine.
+            </p>
+
+            <h5>Uploading a Service Account Key File</h5>
+            <p>
+              There are two ways to authenticate the Google Cloud Monitoring plugin. You can upload a Service Account
+              key file or automatically retrieve credentials from the Google metadata server. The latter option is only
+              available when running Grafana on a GCE virtual machine.
+            </p>
+            <p>
+              The <strong>Monitoring Viewer</strong> role provides all the permissions that Grafana needs. The following
+              API needs to be enabled on GCP for the data source to work:{' '}
+              <a
+                className="external-link"
+                target="_blank"
+                rel="noreferrer"
```

Comment:
this is possibly an overly cautious suggestion, but it might make sense to use noopener as well since it's a target _blank